manager/gui/src/components/LrHeader.vue

```diff
@@ -16,10 +16,10 @@
 -->
 <template>
   <div class="lr-header pa-3">
-    <v-btn text icon color="#000000" @click.stop="$emit('menu-click')">
+    <v-btn text icon color="#fff" @click.stop="$emit('menu-click')">
```

Comment:
Always use 6 digit color code.

Reply:
Switched to all 3 digit color code to 6 digit color code.